test/module/irohad/network/block_loader_test.cpp

```diff
@@ -82,6 +82,15 @@ class BlockLoaderTest : public testing::Test {
         .createdTime(iroha::time::now());
   }
 
+  rxcpp::observable<std::shared_ptr<shared_model::interface::Block>>
+  getBlockObservable(shared_model::proto::Block &block) {
+    return rxcpp::observable<>::create<
```

Comment:
Should be possible to replace with `rxcpp::observable<>::just(block).map([](auto &&x) { return wBlock(x.copy())})`, which seems simpler.